src/Components/Form/FormFields/FormField.tsx

```diff
@@ -71,7 +71,12 @@ const FormField = ({
         )}
       </div>
       <div className={field?.className}>{children}</div>
-      <FieldErrorText error={field?.error} className={field?.errorClassName} />
+      {field?.error && (
+        <FieldErrorText
+          error={field?.error}
+          className={field?.errorClassName}
+        />
+      )}
```

Comment:
This will cause layout shifts when error appears and disappears. You can pass `errroClassName="hidden"` if you know a field won't have any errors instead!

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/LayoutShift

Reply:
Yes but this is standard practise and there really is no way to avoid a little shift here. Always reserving space and make the user of the component manage the classes for how error is rendered inside is not the right way to do this.

It is how all UI libraries do it, MUI and Mantine example below

https://github.com/coronasafe/care_fe/assets/24858182/9d6e1de3-fb68-4326-a25a-8b0b25b2d7cb

Reply:
I do understand that. But other UI libraries having little layout shifts doesn't mean errors in Care UI components should also have layout shifts. 

Also, The consistent spacing below the field is used to provide consistent padding between fields across the platform when multiple fields are in the same column.

Curent changes breaks the consistent padding across the entire platform!

<img width="1822" alt="image" src="https://github.com/coronasafe/care_fe/assets/25143503/2b0bff8f-8158-479e-998d-1ad342476911">

Reply:
Feel free to create a separate issue to address this. This issue is not in the scope of this PR

Reply:
The point of showing how major UI libraries are doing it is to drive across the point that it is the standard way to do things and we should ideally follow standards and learn from these libraries.

Another things these libraries do is not have any kind of margin or spacing on any UI component by default, you have to add your own margin when using the component otherwise there is no spacing, not following this is why the app is breaking in other places. Having the ui component handle margin is not the correct way to do things, any UI component should not apply its own margins as it reduces its ability to be reused. The layout is something the user of the component should manage and apply margin as necessary.

The reason this is really breaking is because it was used incorrectly by depending on that empty error span for the padding which is not ideal.

I see how this is going to be a bigger change that is breaking other stuff in the app so I will leave it out of this PR but this is a refactor that should seriously be considered as a separate issue.